code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -120,6 +119,14 @@
 	last_healed = world.time + heal_interval
 
 /obj/effect/alien/resin/special/pylon/core/attack_alien(mob/living/carbon/Xenomorph/M)
+	if(!warn && world.time < HIVECORE_COOLDOWN_CUTOFF)
+		if((alert(M, "Are you sure that you want to destroy the hive core? (5 minute cooldown before you can build another one.)", , "Yes", "No") == "No"))
```

Comment:
make it clear that destroying it in the early-game will not have the cooldown - use a different alert